rocksdbjni maven build does not escape slashes in groovy script

On Windows `project.basedir` can return a path that contains backslashes as the file separator, but this breaks Java language syntax when it's in the string like that: `"${project.basedir}"`

The error:

```
[ERROR] Failed to execute goal org.codehaus.gmaven:groovy-maven-plugin:2.0:execu
te (default) on project rocksdbjni: Execution default of goal org.codehaus.gmave
n:groovy-maven-plugin:2.0:execute failed: startup failed:
[ERROR] script1.groovy: 1: unexpected char: '\' @ line 1, column 35.
```

--- a/java/rocksjni.pom
+++ b/java/rocksjni.pom
@@ -102,7 +102,7 @@
                                 <name>Xenu</name>
                             </defaults>
                             <source>
-                                String fileContents = new File("${project.basedir}/../include/rocksdb/version.h").getText('UTF-8')
+                                String fileContents = new File(project.basedir.absolutePath + '/../include/rocksdb/version.h').getText('UTF-8')
                                 matcher = (fileContents =~ /(?s).*ROCKSDB_MAJOR ([0-9]+).*?/)
                                 String major_version = matcher.getAt(0).getAt(1)
                                 matcher = (fileContents =~ /(?s).*ROCKSDB_MINOR ([0-9]+).*?/)

